js/options.js

```diff
@@ -38,8 +38,11 @@ function onSave() {
     'reloadUnpinnedOnly': $('reloadUnpinnedOnly').checked,
     'reloadAllLeft': $('reloadAllLeft').checked,
     'reloadAllRight': $('reloadAllRight').checked,
+    'closeAllLeft': $('closeAllLeft').checked,
+    'closeAllRight': $('closeAllRight').checked,
     'reloadStartup': $('reloadStartup').value,
-    'bypassCache': $('bypassCache').checked
+    'bypassCache': $('bypassCache').checked,
+	'buttonDefaultAction': $('buttonDefaultAction').value

```

Comment:
```suggestion
    'buttonDefaultAction': $('buttonDefaultAction').value
```
nit: this is a tab, but the project uses spaces, can you please convert tabs to spaces ? If you are using VSCode, you can change the default behavior for that workspace as well.

Reply:
updated in my fork